code/stata-linebreak.do

```diff
@@ -1,15 +1,20 @@
-* This is GOOD
-    graph hbar ///
-        invil if (priv == 1) ///
-      , over(statename, sort(1) descending) blabel(bar, format(%9.0f)) ///
+GOOD:
+    graph hbar invil /// Proportion in village
+      if (priv == 1) /// Private facilities only
+      , over(statename, sort(1) descending) /// Order states by values
+        blabel(bar, format(%9.0f))          /// Label the bars
         ylab(0 "0%" 25 "25%" 50 "50%" 75 "75%" 100 "100%") ///
         ytit("Share of private primary care visits made in own village")
 
-* This is BAD
+BAD:
     #delimit ;
     graph hbar
         invil if (priv == 1)
       , over(statename, sort(1) descending) blabel(bar, format(%9.0f))
         ylab(0 "0%" 25 "25%" 50 "50%" 75 "75%" 100 "100%")
         ytit("Share of private primary care visits made in own village");
     #delimit cr
+
+UGLY:
+    graph hbar /*
```

Comment:
Hahah. This is both ugly and unnecessarily complicated. who used this?!

Reply:
Truly I have seen this in code review